tests/rptest/services/flink.py

```diff
@@ -373,23 +386,83 @@ def stop_node(self, node):
         # No local node or hostname removing as they needed for log extraction
         return
 
-    def get_active_jobs(self):
+    def get_active_jobs(self, node):
         """
             Get all active jobs from flink REST API
         """
-        jobs = self.list_jobs()
+        jobs = self.list_jobs(node)
         active = [
             job for job in jobs['jobs']
             if job['status'] in self.job_active_statuses
         ]
         return active
 
-    def _has_active_jobs(self):
+    def is_job_idle(self, jobid) -> bool:
```

Comment:
this is _way_ too complicated lol, cant imagine flink API is this horrible, we are probably missing something.